### DISCUSSION TITLE
how to debug when develop a module

### DISCUSSION THREAD

**Discussion by hss01248** [LSPosed/LSPosed#1039] [Q&A] [answered]
i develop a xposed module according to the guide:
https://www.freebuf.com/articles/terminal/189021.html
 but when i install the app into my root phone, 
the module can be recognised by LSPosed,
but the active button can not be enabled.

i have try reboot ,sign the apk, nothing changed.
![image](https://user-images.githubusercontent.com/10666343/131982101-04b31249-6f92-479b-acc5-b3dfd4a7f999.png)

**Comment by hss01248** [comment]
lsposed version: v1.5.2

**Comment by yujincheng08** [comment]
Any log?

**Reply by hss01248** [reply]
no logs output

**Comment by hss01248** [comment, marked as answer]
problem solved, it"s cased by the  gradle version
